acquire/acquire.py

```diff
@@ -2269,6 +2278,18 @@ def main() -> None:
     log.info("Default Arguments: %s", " ".join(args.config.get("arguments")))
     log.info("")
 
+    # start GUI if requested through CLI / config
+    flavour = "CLI"
+    if args.gui == "always" or (args.gui == "depends" and os.environ.get("KEYSOURCE") == "PROMPT" and sys.argc < 3):
```

Comment:
```suggestion
    if args.gui == "always" or (args.gui == "depends" and os.environ.get("PYS_KEYSOURCE") == "PROMPT" and sys.argc < 3):
```
So that it's clear where it came from?

Reply:
This requires a change in PYS as well. Agree?

Reply:
Yes.